packages/modules/backup_clouds/samba/backup_cloud.py

```diff
@@ -8,16 +12,44 @@
 
 log = logging.getLogger(__name__)
 
+def isPortOpen(host, port):
```

Comment:
```suggestion
def is_port_open(host: str, port: int):
```
Anmerkung:
In diesem Fall ist `is port open` passend, da es eine Methode ist. Als Variable wäre `port is open` besser.